src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders character link', () => {
```

Comment:
nice work on the test-- getting it in the correct spot and identifying what you're testing for when the component mounts!